app/components/ui/molecules/DashboardListItem.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { Flex, Box } from 'grid-styled'
+import { differenceInCalendarDays, format } from 'date-fns'
+import { th } from '@pubsweet/ui-toolkit'
+import PropTypes from 'prop-types'
+
+const dashboardStatusColor = (status, theme) =>
+  ({
+    1: theme.colorTextSecondary,
+    2: theme.colorError,
+    3: theme.colorSuccess,
+  }[status])
+
+const dashboardStatusText = status =>
+  ({
+    1: 'Waiting for decision',
+    2: 'Rejected',
+    3: 'Approved',
+  }[status])
+
+const dashboardDateText = date => {
+  const diffDays = differenceInCalendarDays(new Date(), date)
+  if (diffDays < 0) {
+    return 'Invalid date'
+  }
+  if (diffDays === 0) {
+    return 'Today'
+  }
+  if (diffDays === 1) {
+    return 'Yesterday'
+  }
+  return `${diffDays} days ago`
+}
+
+const Root = styled(Flex)`
+  height: ${th('space.5')}
+  align-items: center;
+`
+const StatusBox = styled(Box)`
+  color: ${props => props.color};
+  font-size: small;
```

Comment:
Right now, this seems to compute the font-size as `13px` so everything is slightly smaller than the designs.

You might be better off separating the layout (`Box`) and leave font size, colour, etc to another element (e.g. `Paragraph`)?

How about:
```js
<Box>
  <Paragraph />
</Box>

<Box>
  <Paragraph />
</Box>

<Box>
  <Paragraph />
</Box>
```

Then it should have the right font sizing

Reply:
Yeah we can always change it later if @chugginselifesciences wants :+1: But I think we can safely say that it's not going to be 13px, right? I think it's a good approach to go by eye and use our best guess. But we can still take guidance from the styleguide and make this component conform to one of the things that exist there, right? i.e. 16px (fontSizeBase) or 14px (fontSizeBaseSmall), rather than introducing a new size now.

Reply:
I'll be updating the style guide in InVision a.s.a.p. so hopefully you'll have more info today. But the specific font-size of the status text should be 16px, and should wrap to a 2nd line if needed. However we have changed the status wording following the meeting last week.

In progress submissions will be "In progress" and once submitted will simply stay as "submitted" in an inactive state.